package/build.gradle

```diff
@@ -0,0 +1,27 @@
+description = 'Sparkling Water Spark Package'
+
+apply plugin: 'com.github.johnrengelman.shadow'
+
+dependencies {
+    // hack to go around Spark Ivy resolver
+    compileOnly "no.priv.garshol.duke:duke:1.2"
+
+    compile project(':sparkling-water-core')
+    compile project(":sparkling-water-examples")
+    compile project(":sparkling-water-ml")
+    compile project(":sparkling-water-repl")
+}
+
+jar {
+    enabled = true
+}
+
+shadowJar {
+    archiveName = jar.archiveName
```

Comment:
@mmalohlava Should we also shadow the jetty used in H2O as we do in regular assembly ?

```  relocate 'org.eclipse.jetty', 'ai.h2o.org.eclipse.jetty'```. This would resolve the jetty issues on Databribrix [SW-236] and when using sw as spark package